druid/examples/game_of_life.rs

```diff
@@ -57,7 +57,6 @@ impl Grid {
                 // death by loneliness or overcrowding
                 if life && (n_lives_around < 2 || n_lives_around > 3) {
                     indices_to_mutate.push(pos);
-                    continue;
```

Comment:
Whilst it is true that the code will perform identically without this `continue`, keeping it in might communicate intent better. What do you think?

Reply:
oh I actually thought it did the opposite, having a break/continue within a loop makes it harder to reason about.
After looking at the idiomatic implementation (https://rustwasm.github.io/book/game-of-life/implementing.html) I think using a match might actually be easiest to reason about?